src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -39,16 +44,27 @@ class Lambda extends RequestStreamHandler {
       asset <- IO.fromOption(assetItems.headOption)(
         new Exception(s"No asset found for ${input.id} and ${input.batchId}")
       )
+      fileReference = asset.identifiers.find(_.identifierName == "BornDigitalRef").map(_.value).orNull
+      log = logger.info(Map("batchRef" -> input.batchId, "fileReference" -> fileReference))(_)
```

Comment:
Consider renaming file reference, maybe follow what we've done in Preservica to `BornDigitalRef`? We should be clear that `asset != file` within our system.

Reply:
As we said, this is only for monitoring so it's probably fine.